source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,356 @@
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+Migration generally has similar effects as :ref:`terminating <terminating-realm-sync>` 
+and re-enabling Device Sync. There are some additional effects of migration:
+
+- In the App Services UI, the Rules and Schema pages will be read-only.
+- Your App will experience a period of read-only Sync and then a 
+  period of downtime.
+- The new Flexible Sync configuration will set 
+  :ref:`client max offline time <client-maximum-offline-time>` to 30 days.
+- Storage in your Atlas cluster used by Device Sync metadata is temporarily doubled.
+  Partition-Based Sync metadata and Flexible Sync metadata must temporarily co-exist. 
+  Depending on the size of your synced data, this could affect billing every time you
+  migrate.
+
+**To migrate from Partition-Based Sync to Flexible Sync:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, select :guilabel:`[Migrate to Flexible Sync]`.
+   
+   .. image:: /images/begin-migration.png
+      :alt: Begin a migration
+   
+#. Read the information about migrating, then select 
+   :guilabel:`Next: Migration Requirements`.
+#. Ensure you meet the migration requirements, the select 
+   :guilabel:`Start Migration`.
+
+.. note:: Temporary Sync Disruption
+
+   Clients will not be able to sync their local writes until the migration has
+   completed.
+
+After a migration is completed, Partition-Based Sync client apps can communicate
+with your App's Flexible Sync backend. You can disable and re-enable Flexible Sync
+and client apps will still communicate with your backend.
+
+If you re-enable Partitition-Based Sync you will lose the "migrated" status.
+This means you'll need to to another migration if you want to use Flexible Sync
+on the backend with Partition-Based Sync client apps.
+
+Migrate Permissions and Rules
+`````````````````````````````
+
+Your App's permissions and rules will be automatically migrated if the Partition-Based
+Sync rules have direct App Services Rules equivalents. This will override
+previously-defined App Services Rules.
+
+In step 4 above, you will see if your permissions and rules can be automatically
+migrated. On the :guilabel:`Review Migration Process` step in the App Services UI,
+look for :guilabel:`Rules & Permisssions`.
+
+Some Partition-Based Sync rule strategies can't translate directly to App Services
+Rules. You may need to manually migrate permissions that include:
+
+- :json-operator:`%function` operator.
+
+  :ref:`Function rules <pbs-function-rules>` are not compatible with Flexible Sync 
+  and cannot be migrated.
+
+- :json-expansion:`%%partition` expansion.
+
+  App Services Rules do not have an equivalent expansion for  
+  :json-expansion:`%%partition`, so it cannot be migrated.
```

Comment:
we actually do migrate this automatically if it's not used in conjunction with the other restrictions listed here.
e.g. `{ %%partition: %%user.id }` becomes `{ <partitionKey>: %%user.id }`

if they need to manually update their permissions, then this is implied in the "See the list of Flexible Sync-compatible expansions for all supported expansions." note below so we can probably remove the bullet here, or change up the wording/move it elsewhere in this section